Bio/SeqUtils/MeltingTemp.py

```diff
@@ -1035,17 +1124,19 @@ def Tm_staluc(s, dnac=50, saltc=50, rna=0):
     # Original method was by Sebastian Bassi <sbassi@genesdigitales.com>. It is
     # now superseded by Tm_NN.
 
-    warnings.warn("Tm_staluc may be depreciated in the future. Use Tm_NN " +
-                  "instead.", PendingDeprecationWarning)
+    warnings.warn(
+        "Tm_staluc may be depreciated in the future. Use Tm_NN " + "instead.",
+        PendingDeprecationWarning,
+    )
     if not rna:
         return Tm_NN(s, dnac1=dnac / 2.0, dnac2=dnac / 2.0, Na=saltc)
     elif rna == 1:
-        return Tm_NN(s, dnac1=dnac / 2.0, dnac2=dnac / 2.0, Na=saltc,
-                     nn_table=RNA_NN2)
+        return Tm_NN(s, dnac1=dnac / 2.0, dnac2=dnac / 2.0, Na=saltc, nn_table=RNA_NN2)
     else:
         raise ValueError("rna={0} not supported".format(rna))
 
 
 if __name__ == "__main__":
     from Bio._utils import run_doctest
+
```

Comment:
Again, is this blank line introduce by black? If not, I'd removed it, otherwise OK

Reply:
Added by black (see above)